pkg/ebpf/processor_funcs.go

```diff
@@ -331,6 +331,12 @@ func (t *Tracee) processPrintMemDump(event *trace.Event) error {
 // normalizeEventCtxTimes normalizes the event context timings to be relative to tracee start time
 // or current time in nanoseconds.
 func (t *Tracee) normalizeEventCtxTimes(event *trace.Event) error {
+	eventId := events.ID(event.EventID)
+	if eventId > events.MaxCommonID && eventId < events.MaxUserSpace {
```

Comment:
nitpick: and we extract for this with better naming? `events.IsDerivedEvent()`
wdyt?

Reply:
> nitpick: can we extract for this with better naming? `events.IsDerivedEvent()`
> wdyt?

LGTM as well. The method suggested is a good idea.

Reply:
I think that when we do extensions, we will need specified "sections" per event type. Then we can easily classify by section location. I assume that that change will break this one, so we don't necessarily need this right now.